src/olympia/zadmin/models.py

```diff
@@ -51,7 +51,7 @@ class ValidationJob(ModelBase):
                                          related_name='validation_current_set')
     target_version = models.ForeignKey(AppVersion,
                                        related_name='validation_target_set')
-    finish_email = models.EmailField(null=True)
+    finish_email = models.EmailField(null=True, max_length=75)
```

Comment:
that was the default length of the `EmailField` in Django 1.7 and earlier, they increased that value in 1.8 but I'd like to do that separately from the general 1.8 upgrade.